pre_commit/clientlib.py

```diff
@@ -222,6 +235,10 @@ def _entry(modname):
     ),
 
     MigrateShaToRev(),
+    cfgv.WarnAdditionalKeys(
+        {'repo', 'rev', 'hooks'},
```

Comment:
let's use a tuple here instead

Reply:
why tuple?

Reply:
immutability, WarnAdditionalKeys takes any `Sequence` and will convert it to a set when comparing the keys